src/views/core/members/UpdateProfile.vue

```diff
@@ -76,13 +76,28 @@
           </div>
         </div>
       </div>
+
+      <template v-if="validationErrors.privacy">
+        <hr/>
+
+        <div class="field">
+          <label class="label">I agree to the <router-link :to="{ name: 'oms.legal.simple' }">Privacy Policy</router-link>
```

Comment:
Actually we should go to { name: 'oms.confluence', params: { page_id: 'terms-of-service' } }
Can you change this for all instances? And while you're at it, also remove the LegalSimple and LegalFull pages

Reply:
Just did that in the newest commit